CHANGES/479.feature

```diff
@@ -0,0 +1,2 @@
+Replaced `artifacts` by `repository_version` as the parameter to provide the build context
+for a Containerfile.
```

Comment:
What kind of `repository_version`? pulp-container's one? 😜

Reply:
Oops ... good catch!